docs/HowToFAQ/Errors-Resolution.md

```diff
@@ -483,9 +534,12 @@ But a ghost context can also be implicit, and not so obvious: if something ghost
 such as the condition of an `if` statement or loop or the expression being matched in a match 
 statement, then the rest of the statement may be required to be ghost.
 
-<!-- TODO: comment on tracking of printeffects -->
+In addition, methods must be marked with the `{:print}` attribute if 
+`--track-print-effects` is enabled and it has 
+`print` statements or calls methods marked with `{:print}`.
```

Comment:
```suggestion
In addition, methods must be marked with the `{:print}` attribute if 
it has `print` statements or calls methods marked with `{:print}`
and `--track-print-effects` is enabled.
```

Reads better IMO to have the "it" closer to the subject.

Reply:
FIne - though 'they' is better